topics/Tutorials.md

```diff
@@ -55,7 +55,8 @@
 * [You, Me & SVG - Code School](https://www.codeschool.com/courses/you-me-svg)
 * [Tools And Resources For Editing, Converting And Optimizing SVGs](https://www.smashingmagazine.com/2016/04/tools-and-resources-for-editing-converting-and-optimizing-svgs/)
 * [SVG animation research & GSAP](http://codepen.io/FabioG/post/svg-animation-research-gsap)
-* [Make Responsive SVG Graphics and Infographics](https://www.sitepoint.com/make-responsive-svg-graphs-infographics)
+* [Make Responsive SVG Graphics and Infographics](https://www.sitepoint.com/make-responsive-svg-graphs-infographics)[
+* [SVG has more potential](https://madebymike.com.au/writing/svg-has-more-potential/) - SVG has more potential than we tend to think and how it provides clever solutions to everyday use cases.
```

Comment:
To be similar with the others links, please, don't put description yet.
I'm not maintaining this anymore, but if you want to help, talk with @jjaderg , he's responsible for the repo now.